requirements.txt

```diff
@@ -1,2 +1,3 @@
 pytest
 pytest-describe
+git+git://github.com/morganstanley/testplan@latest#egg=testplan
```

Comment:
It looks like removing python restriction breaks the build for other python versions. Are there any reasons why you've removed it?

Reply:
just to speed up the CI result, I'll revert it when I found the issue.